src/librustc/hir/lowering.rs

```diff
@@ -58,6 +58,7 @@ use std::mem;
 use smallvec::SmallVec;
 use syntax::attr;
 use syntax::ast;
+use syntax::ptr::P as AstP;
```

Comment:
There is a `syntax::ptr::P` in this file IIRC, that you might want to change to `AstP`.

Reply:
Thanks! Changed it.